lib/app/config/logger.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:logger/logger.dart';
+
+final logger = Logger();
```

Comment:
@alexdivadi usually what i do is make a thin wrapper so you have static Log.d and Log.e and Log.e can take an exception.  This allows 1) If you switch logging libraries you have that tiny abstraction layer and you're not stuck with one 2) more so, you have a better shot at security and remote features for production, ie
1) release mode in most cases should not have much logging to the console for security and best practice, esp in prod
2) release mode should have remote logging and crash reporting ie crashlytics

just today i was trying to explain (wiht minimal success) how crashlytics seems to handle better if you have an exception not just a message - but you definitely want the exception if it is there.

Reply:
So should I make the logger a riverpod provider? @neiljaywarner